jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java

```diff
@@ -227,7 +227,7 @@ public void testNonDefaultsWriterFactory() throws ValidationException {
 		assertThat(
 			gelfWriterFactory.getGelfConfiguration().getTlsTrustCertChainFile()
 				.getPath()
-		).isEqualTo("/somefile")
+		).isEqualTo(File.separator + "somefile")
```

Comment:
Even better would probably to compare File object. I wonder if AssertJ has some support to do this more cleanly...